hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -598,6 +598,15 @@ public Config addRingBufferConfig(RingbufferConfig ringbufferConfig) {
         return ringbufferConfigs;
     }
 
+    public Config setRingbufferConfigs(Map<String, RingbufferConfig> ringbufferConfigs) {
```

Comment:
Spring expects the setters since the SpringXmlBuilderHelper#configBuilder declares nested configurations for all ManagedMaps by strings (see SpringXmlConfigBuilder).